### ISSUE TITLE
Accessibility: Link Name for Home nav link

### ISSUE THREAD

**Issue by gcharest** [sara-sabr/ITStrategy#649]
Ran a quick accessibility test with Accessibility Insights for Web on Chrome and found the following failed test:

Path | .navbar-brand
-- | --

`<a class="navbar-brand" href="/ITStrategy/accueil.html">`

Fix the following:

- Element is in tab order and does not have accessible text

Fix ONE of the following:

- Element does not have text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element's default semantics were not overridden with role="presentation"
- Element's default semantics were not overridden with role="none"

**Comment by Moro-Code** [comment]
On it... thanks for the check

**Comment by Moro-Code** [comment]
resolved by pull  #656
